### PR TITLE
added more scale objects and helper file

### PR DESCRIPTION
**Pull request by micahnyc**

### CODE REVIEW

**Comment by micahnyc** [comment]
https://drive.google.com/file/d/1bXBMmSix9i2kAIPUy5axBLK77KAfMS7V/view?usp=share_link

golden_gate and burj .osmodel files need to be updated but wanted to put up PR in the meantime

File: data/assets/util/scale_model_helper.asset [hunk 1]
```diff
@@ -0,0 +1,111 @@
+local earthAsset = asset.require('scene/solarsystem/planets/earth/earth')
```
Comment (alexanderbock) [inline data/assets/util/scale_model_helper.asset:1]: I don't think this file should exist;  it increases the difficulty for a novice builder to go in and understand and replicate the scale behavior for their own model.  I think the contents of this file should be at the top of the each of the new scale models instead where a new user can exchange the `syncedResource` for a `localResource` and still use the other ones.
It makes it unnecessarily complicated to understand the "educational" assets

File: data/assets/educational/scale/arc_de_triomphe.asset [hunk 2]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Arc de Triomphe"
+local identifier = "ScaleArcDeTriomphe"
```
Comment (alexanderbock) [inline data/assets/educational/scale/arc_de_triomphe.asset:4]: -> `Scale_ArcDeTriomphe`?

File: data/assets/educational/scale/big_ben.asset [hunk 3]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Big Ben"
+local identifier = "ScaleBigBen"
```
Comment (alexanderbock) [inline data/assets/educational/scale/big_ben.asset:4]: -> `Scale_BigBen`

File: data/assets/educational/scale/burj_khalifa.asset [hunk 4]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Burj Khalifa"
+local identifier = "ScaleBurjKhalifa"
```
Comment (alexanderbock) [inline data/assets/educational/scale/burj_khalifa.asset:4]: -> `Scale_BurjKhalifa`

File: data/assets/educational/scale/empire_state_building.asset [hunk 5]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Empire State Building"
+local identifier = "ScaleEmpireStateBuilding"
```
Comment (alexanderbock) [inline data/assets/educational/scale/empire_state_building.asset:4]: -> `Scale_EmpireStateBuilding`

File: data/assets/educational/scale/gateway_arch.asset [hunk 6]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Gateway Arch"
+local identifier = "ScaleGatewayArch"
```
Comment (alexanderbock) [inline data/assets/educational/scale/gateway_arch.asset:4]: -> `Scale_GatewayArch`

File: data/assets/educational/scale/golden_gate_bridge.asset [hunk 7]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Golden Gate Bridge"
+local identifier = "ScaleGoldenGateBridge"
```
Comment (alexanderbock) [inline data/assets/educational/scale/golden_gate_bridge.asset:4]: -> `Scale_GoldenGateBridge`

File: data/assets/educational/scale/kuala_lumpur_tower.asset [hunk 8]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Kuala Lumpur Tower"
+local identifier = "ScaleKualaLumpurTower"
```
Comment (alexanderbock) [inline data/assets/educational/scale/kuala_lumpur_tower.asset:4]: -> `Scale_KualaLumpurTower`

File: data/assets/educational/scale/rose_bowl.asset [hunk 9]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Rose Bowl"
+local identifier = "ScaleRoseBowl"
```
Comment (alexanderbock) [inline data/assets/educational/scale/rose_bowl.asset:4]: -> `Scale_RoseBowl`

File: data/assets/educational/scale/school_bus.asset [hunk 10]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "School bus"
+local identifier = "ScaleSchoolBus"
```
Comment (alexanderbock) [inline data/assets/educational/scale/school_bus.asset:4]: -> `Scale_SchoolBus`

File: data/assets/educational/scale/scientist.asset [hunk 11]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Scientist"
+local identifier = "ScaleScientist"
```
Comment (alexanderbock) [inline data/assets/educational/scale/scientist.asset:4]: -> `Scale_Scientist`

File: data/assets/educational/scale/statue_of_liberty.asset [hunk 12]
```diff
@@ -0,0 +1,32 @@
+local helper = asset.require('util/scale_model_helper')
+
+local name = "Statue of Liberty"
+local identifier = "ScaleStatueOfLiberty"
```
Comment (alexanderbock) [inline data/assets/educational/scale/statue_of_liberty.asset:4]: -> `Scale_StatueOfLiberty`

File: data/assets/educational/scale/gateway_arch.asset [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+local lat = 38.6248806710205
+local lon = -90.18493947620934
+local altitude = 0
+local scale = 0.5
+local rotationVector = {0.0, 157.0, 0.0}
```
Comment (alexanderbock) [inline data/assets/educational/scale/gateway_arch.asset:9]: -> `{ 0.0, 157.0, 0.0 }`

File: data/assets/educational/scale/golden_gate_bridge.asset [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+local lat = 37.82011483830592
+local lon = -122.47789030197121
+local altitude = 0
+local scale = 0.5
+local rotationVector = {0.0, 157.0, 0.0}
```
Comment (alexanderbock) [inline data/assets/educational/scale/golden_gate_bridge.asset:9]: -> `{ 0.0, 157.0, 0.0 }`

File: data/assets/educational/scale/kuala_lumpur_tower.asset [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+local lat = 3.152878
+local lon = 101.703533
+local altitude = 0
+local scale = 1
+local rotationVector = {0.0, 98.68, 0.0}
```
Comment (alexanderbock) [inline data/assets/educational/scale/kuala_lumpur_tower.asset:9]: -> `{ 0.0, 98.68, 0.0 }`

File: data/assets/educational/scale/rose_bowl.asset [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+local lat = 34.161318
+local lon = -118.167623
+local altitude = 0
+local scale = 1
+local rotationVector = {0.0, 98.68, 0.0}
```
Comment (alexanderbock) [inline data/assets/educational/scale/rose_bowl.asset:9]: -> `{ 0.0, 98.68, 0.0 }`

File: data/assets/educational/scale/school_bus.asset [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+local lat = 40.780671
+local lon = -73.971701
+local altitude = 0
+local scale = 1
+local rotationVector = {0.0, 90.0, 0.0}
```
Comment (alexanderbock) [inline data/assets/educational/scale/school_bus.asset:9]: -> `{ 0.0, 90.0, 0.0 }`

File: data/assets/educational/scale/scientist.asset [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+local lat = 34.201639
+local lon = -118.171319
+local altitude = 0.78
+local scale = 1
+local rotationVector = {0.0, 90.0, 0.0}
```
Comment (alexanderbock) [inline data/assets/educational/scale/scientist.asset:9]: -> `{ 0.0, 90.0, 0.0 }`

**Comment by alexanderbock** [comment]
Would be nice for the Gateway Arch to not have the surface vertices as it looks kind of weird
image

File: data/assets/util/scale_model_helper.asset (hunk 1 above)
Comment (micahnyc) [reply data/assets/util/scale_model_helper.asset:1]: I don't want to paste 70 lines of code into 10 files just to make their code easier to understand. 

Here "educational" is meant to infer that the assets are used for educating the audience, not educational on how to create assets

**Comment by micahnyc** [comment]
golden_gate, gateway arch, and burj .osmodel still to be updated after prerelease

**Pull request merged**